docs/BUILD_PIPELINE_DEBUGGING.md

```diff
@@ -0,0 +1,30 @@
+# Build Pipeline Debugging
```

Comment:
I think this should be cross linked in https://github.com/ember-cli/ember-cli/blob/master/PERF_GUIDE.md